src/components/button.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
```

Comment:
I've never done anything with React before, but are semicolons still required? Website works well locally though

Reply:
Don't think so as it gets converted into _good_ js when gatsby builds the site.

Reply:
Yeah.. can you turn on semicolons? If you disable semicolons, it may lead to bugs because of javascript. See https://www.freecodecamp.org/news/codebyte-why-are-explicit-semicolons-important-in-javascript-49550bea0b82/

Reply:
A good read, never looked at semicolons that way before.